src/main/java/net/neoforged/neoforge/event/level/ExplosionEvent.java

```diff
@@ -80,4 +81,42 @@ public List<Entity> getAffectedEntities() {
             return entityList;
         }
     }
+
+    /**
+     * ExplosionEvent.Knockback is fired once the explosion has calculated the knockback velocity to give to the entity caught in blast.<br>
+     * <br>
+     * This event is not {@link ICancellableEvent}.<br>
+     * This event does not use {@link HasResult}.<br>
+     * This event is fired on the {@link NeoForge#EVENT_BUS}.<br>
+     */
+    public static class Knockback extends ExplosionEvent {
```

Comment:
The event should be in its own standalone class file as `ExplosionKnockbackEvent`